bitwindow/server/main.go

```diff
@@ -80,8 +81,14 @@ func realMain(ctx context.Context, cancelCtx context.CancelFunc) error {
 	}
 	defer database.SafeDefer(ctx, db.Close)
 
-	bitcoindConnector := func(ctx context.Context) (bitcoindrpc.BitcoinServiceClient, error) {
-		return startCoreProxy(ctx, conf)
+	bitcoindConnector := func(ctx context.Context) (corerpc.BitcoinServiceClient, error) {
+		bitcoind, err := startCoreProxy(ctx, conf)
+		return bitcoind, err
+	}
+
+	coreProxy, err := startCoreProxy(ctx, conf)
+	if err != nil {
+		return fmt.Errorf("init core proxy: %w", err)
 	}
```

Comment:
This makes bitwindow depend on core to start? We don't want that

Reply:
Good catch, fixed!